serverless-functions/src/functions/features/park-interaction/flex/park-interaction.js

```diff
@@ -106,8 +85,36 @@ exports.handler = prepareFlexFunction(requiredParameters, async (context, event,
               webhookSid: webhookResult.webhook.sid,
             },
           });
+          createMapItem = createMapItemResponse.mapItem;
         }
       }
+
+      // update conversation attributes
+      const attributes = {
+        interactionSid,
+        channelSid,
+        participantSid,
+        taskSid,
+        workflowSid,
+        taskChannelUniqueName,
+        queueName,
+        queueSid,
+        workerSid,
+        taskAttributes,
+        webhookSid: webhookResult.webhook.sid,
+      };
+
+      if (createUpdateSyncMapItem && createMapItem.mapSid && createMapItem.key) {
+        attributes.mapSid = createMapItem.mapSid;
+        attributes.mapItemKey = createMapItem.key;
+      }
+
+      const updateAttributesResponse = await ConversationsOperations.updateAttributes({
+        context,
+        conversationSid,
+        attributes: JSON.stringify(attributes),
+      });
+      if (!updateAttributesResponse.success) throw updateAttributesResponse.message;
```

Comment:
Now updating conversation attributes later
And adding these new attributes if a sync map item was created